Workbooks/SapMonitor/SapHanaInfrastructure/SapHanaInfrastructure.workbook

```diff
@@ -0,0 +1,376 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "SapHana_LoadHistory_CL\r\n|summarize arg_max(UTC_TIMESTAMP_t, *) by HOST_s\r\n|project Host=HOST_s , [\"CPU Utilization (%)\"]=CPU_d ,  [\"Memory Utilization (%)\"]=round((MEMORY_USED_d/MEMORY_SIZE_d)*100,2),[\"Disk Utilization (GB)\"]=DISK_USED_d  , [\"Network In\"]=NETWORK_IN_d , [\"Network Out\"]=NETWORK_OUT_d ",
```

Comment:
also, is `UTC_TIMESTAMP_t` the standard time column in your tables?  the code that figures out if a time range is specified in a query is looking for specific known column names (like `timestamp` in app insights and `TimeGenerated` in most log analytics tables.  do i need to add this to our list?

Reply:
As I'm batch-ingesting the collected metrics using Log Analytics's [Data Collector API](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/data-collector-api), I went for a separate custom field `UTC_TIMESTAMP_t` for each record. (I understand there is a way to overwrite the TimeGenerated field by setting the `time-generated-field` HTTP header, but that would limit me to one row per HTTP ingestion call.)

So yes, please add UTC_TIMESTAMP_t to your list.